docker-compose.yml

```diff
@@ -1,79 +1,119 @@
 # This is example of docker-compose file
 # which can be used to run NexentaFusion ecosystem.
-# 
-# Please note, that following settings provide
-# password protected Elasticsearch (also with SSL enabled)
-# Check elasticsearch environment settings:
-#   AUTH_USERNAME
-#   AUTH_PASSWORD
-#
-# Make sure your docker host machine vm.max_map_count kernel setting value is at least 262144. 
-# It is necessary for the Elasticsearch container. 
-# Here is an instruction how to get and set this kernel setting 
-# https://www.elastic.co/guide/en/elasticsearch/reference/7.2/docker.html#docker-cli-run-prod-mode
 #
 # Usage:
-#   docker-compose up -d
+#   docker-compose up
+
+version: '3.4'
+
+# Database shared environment
+x-elasticsearch-env: &elasticsearch-env
+  # Heap size
+  ES_JAVA_OPTS: "-Xms1g -Xmx1g"
+
+  # Cluster name
+  cluster.name: "fusion-docker-cluster"
 
-version: '3.2'
+  # Default settings for all cluster nodes
+  bootstrap.memory_lock: "true"
+  cluster.initial_master_nodes: "es-master01,es-master02,es-master03"
 
 services:
   fusion:
-    image: nexenta/fusion-dev:${VERSION:-2.0.0.3}
+    image: tintri-dockerv2-local.jfrog.io/fusion2_develop:${VERSION:-latest}
```

Comment:
Is it okay for customer-facing file? This is our internal use repo.

Reply:
I've added a TODO comment to not forget to change the link to image. Certainly we must not use the internal images. But since there is no public ones yet, we can use internal to test the docker-compose file.